testgrid/config.yaml

```diff
@@ -3214,32 +3214,32 @@ test_groups:
   num_failures_to_alert: 1
 
 # Gardener Test Groups
-- name: ci-gardener-e2e-conformance-aws-v1.11
-  gcs_prefix: k8s-conformance-gardener/ci-gardener-e2e-conformance-aws-v1.11
+- name: ci-gardener-e2e-conformance-aws-v1.13
```

Comment:
will note to come back and document later but I know we have major time-zone clashes so will go ahead and get this in - to confirm these are 1.13 stable, as in 1.13.2, or 1.13.X under development (IE build from release-1.13 at HEAD) ?

Reply:
Hi @BenTheElder, it's 1.13.2 version.

Reply:
thanks for clarifying, at some point I want to further organize how we document this :+1: